### PR TITLE
Add email and sms confirmation actions

### PR DESCRIPTION
**Pull request by sinclairs**
* applicant:adult
* mainapplicant.applicant:adult:incapable
* mainapplicant.applicant:child
* rep.applicant:adult:capable
* rep.mainapplicant.applicant:adult:incapable
* rep.mainapplicant.applicant:child
* rep:no-legal-authority.applicant

### CODE REVIEW

File: src/template.js [hunk 1]
```diff
@@ -710,80 +710,254 @@ module.exports = {
...
-                    l10n: {
-                        vars: {
-                            lng: 'en',
-                            context: {
-                                $data:
-                                    '/answers/p-applicant-who-are-you-applying-for/q-applicant-who-are-you-applying-for'
-                            },
-                            ns: 'notification-confirmation'
+                    // prettier-ignore
+                    cond: ['==', '$.answers.p-applicant-confirmation-method.q-applicant-confirmation-method', 'email'],
```
Comment (armoj) [inline src/template.js:718]: I can see why this just 1 condition, but would it be worth being explicit and have a condition testing for not being a `proxy`?

File: src/template.js [hunk 2]
```diff
@@ -710,80 +710,254 @@ module.exports = {
...
                         personalisation: {
                             caseReference: '||/answers/system/case-reference||'
                         },
-                        reference: null // required for notify api
+                        reference: null
                     }
                 },
-                sendSms: {
+                {
+                    description: 'Confirmation email - rep.mainapplicant.applicant:adult:incapable',
```
Comment (armoj) [inline src/template.js:784]: should we add the `authority` label to this description to be explicit, and not infer it?

`Confirmation email - rep.mainapplicant:authority.applicant:adult:incapable`

File: src/template.js [hunk 3]
```diff
@@ -710,80 +710,254 @@ module.exports = {
...
+                            '||/answers/p-rep-confirmation-method/q-rep-enter-your-telephone-number||',
                         personalisation: {
                             caseReference: '||/answers/system/case-reference||'
                         },
-                        reference: null // required for notify api
+                        reference: null
+                    }
+                },
+                {
+                    description: 'Confirmation sms - rep.mainapplicant.applicant:adult:incapable',
```
Comment (armoj) [inline src/template.js:907]: should we add the `authority` label to this description to be explicit, and not infer it?

`Confirmation sms - rep.mainapplicant:authority.applicant:adult:incapable`

**Review by armoj — CHANGES_REQUESTED** [review]

File: src/template.js (hunk 1 above)
Comment (sinclairs) [reply src/template.js:718]: Probably a good shout. Nothing wrong with being more explicit. Given this has been through testing, you ok if we do this at a later date?

File: src/template.js (hunk 2 above)
Comment (sinclairs) [reply src/template.js:784]: Will check with Julie on her return

**Pull request merged**